### PR TITLE
Adds documentation for actionkit signup

### PR DESCRIPTION
**Pull request by shakalee14**

### CODE REVIEW

**Review by schuyler1d — APPROVED** [review]
:+1: 
My one nit/suggestion is to remove the 'Step One'/Step Two, etc -- since they're not really steps, but that's just my read -- feel free to ignore.

File: docs/HOWTO_INTEGRATE_WITH_ACTIONKIT.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+- AK_SECRET should be your secret from your ActionKit account
+
+## Step Two - making sure you have the correct data in your contacts list
+
+- In order for ActionKit RSVP Syncing to work with Spoke, we have to make sure we're including the correct information in our CSV upload AND set up the interaction steps correctly in the campaign script.
+
+### Uploading CSVs with the Correct Information
+
+- When you upload a CSV to Spoke, you must make sure you have the following fields filled out for each contact:
+  * external_id (some reference to an id of the user in actionkit)
```
Comment (mathemagica) [inline docs/HOWTO_INTEGRATE_WITH_ACTIONKIT.md]: it'd be helpful to specify whether this is the akid (usually found in the url) or the actionkit user_id (found in the core_user table in the actionkit database)

File: docs/HOWTO_INTEGRATE_WITH_ACTIONKIT.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+- The event_id refers to the event id found in AK. The easiest way to find this is in the url slug for the event page
+- The event_page refers to the page name in action kit. The slug in the url in the AK event page is sometimes unreliable/confusing
+- In order for AK event syncing to work, external_id, event_page and event_id need to be correct.
+
+## Step Three - creating a script linked to ActionKit Event Signup
+
+- After uploading your csv to Spoke, you will assign texters and then create an interaction script for your texters.
+
+- In the interaction script, After you fill out the initial Script and Question fields, more options will appear for possible answers for your texters to mark.
+- Make sure to mark "Yes" as an ActionKit Event RSVP. When the texter then receives a yes reply from a member, the texter will choose yes, and that yes will be "sent" to ActionKit for that event.
```
Comment (mathemagica) [inline docs/HOWTO_INTEGRATE_WITH_ACTIONKIT.md]: "Make sure to mark "Yes" as an ActionKit Event RSVP." -> Does "Yes" have to be an answer option? and is it case sensitive?

Comment (mathemagica) [reply docs/HOWTO_INTEGRATE_WITH_ACTIONKIT.md]: "will be sent to ActionKit for that event." -- is it worth clarifying here that "sent" means an API request to ActionKit, and that testers can check their spoke server logs to see the API request go out to make sure its working?

**Review by mathemagica — CHANGES_REQUESTED** [review]

**Pull request merged**